app/src/lib/drag-and-drop-manager.ts

```diff
@@ -48,11 +52,12 @@ export class DragAndDropManager {
 
   public dragStarted(): void {
     this._isDragInProgress = true
+    this.emitter.emit('drag-started', {})
   }
 
   public dragEnded(dropTargetSelector: DropTargetSelector | undefined) {
-    this.emitter.emit('drag-ended', dropTargetSelector)
     this._isDragInProgress = false
+    this.emitter.emit('drag-ended', dropTargetSelector)
```

Comment:
Hopefully this change in the order (which I think makes more sense: notify drag has ended when internally is marked as such) doesn't affect anything. I've tested cherry-picking and squash and all seemed to work as expected.

Reply:
Makes sense to me